docs/conf/portland/2018/sponsors/prospectus.rst

```diff
@@ -137,11 +144,14 @@ The **Patron** package highlights your company as a force in the industry and co
 - Name included in welcome announcement in email newsletters and social media
 - Display 3 promotional (“Swag”) item on the conference swag table (provided by sponsor)
 - Three featured job postings in our Newsletter (2,500 subscribers)
+- A featured booth at the job fair
```

Comment:
we can't call these booths, and then say "we don't do booths". How about table, featured table? 

Should we describe the difference between table / featured table?

Reply:
+1 for using "table" instead of "booth" (should be fixed everywhere if accepted)